packages/xo-web/src/xo-app/new/network/index.js

```diff
@@ -0,0 +1,257 @@
+import _, { messages } from 'intl'
+import ActionButton from 'action-button'
+import decorate from 'apply-decorators'
+import Page from '../../page'
+import PropTypes from 'prop-types'
+import styles from './index.css'
+import React, { Component } from 'react'
+import Wizard, { Section } from 'wizard'
+import { map } from 'lodash'
+import { Toggle } from 'form'
+import { connectStore } from 'utils'
+import { injectIntl } from 'react-intl'
+import { linkState } from 'reaclette-utils'
+import { Container, Row, Col } from 'grid'
+import { injectState, provideState } from 'reaclette'
+import { SelectPif, SelectPool } from 'select-objects'
+import { createGetObject, getIsPoolAdmin } from 'selectors'
+import { createBondedNetwork, createNetwork, getBondModes } from 'xo'
+
+const getInitialState = () => ({
+  bonded: false,
+  bondMode: undefined,
+  description: '',
+  mtu: '',
+  name: '',
+  pif: undefined,
+  pifs: undefined,
+  vlan: '',
+})
+
+const NewNetwork = decorate([
+  connectStore(() => ({
+    isPoolAdmin: getIsPoolAdmin,
+    pool: createGetObject((_, props) => props.location.query.pool),
+  })),
+  injectIntl,
+  provideState({
+    initialState: () => ({ ...getInitialState(), bondModes: undefined }),
```

Comment:
`bondModes`:  to avoid losing its value after` reset` or `onChangeBondedNetwork`.